src/pytorch_lightning/strategies/launchers/multiprocessing.py

```diff
@@ -95,12 +95,12 @@ def launch(self, function: Callable, *args: Any, trainer: Optional["pl.Trainer"]
             nprocs=self._strategy.num_processes,
```

Comment:
The docstring of `{_MultiProcessingLauncher,_XLALauncher}.launch()` needs changing, too.
```diff
-       """Spawns processes that run the given function in parallel.
+       """Launches processes that run the given function in parallel.

        The function is allowed to have a return value. However, when all processes join, only the return value
        of worker process 0 gets returned from this `launch` method in the main process.

        Arguments:
-           function: The entry point for all spawned processes.
+           function: The entry point for all launched processes.
            *args: Optional positional arguments to be passed to the given function.
            trainer: Optional reference to the :class:`~pytorch_lightning.trainer.trainer.Trainer` for which
                a selected set of attributes get restored in the main process after processes join.
            **kwargs: Optional keyword arguments to be passed to the given function.
        """
```